README.md

```diff
@@ -32,10 +32,11 @@ contract ExampleContract {
     }
 
     function getBTCUSDPrice(bytes[] memory priceUpdateData) public returns (PythStructs.Price memory) {
-        // Update the prices to be set to the latest values. The `priceUpdateData` data should be
-        // retrieved from our off-chain Price Service API using the `pyth-evm-js` package.
+        // Update the prices to the latest available values and pay the required fee for it. The `priceUpdateData` data
+        // should be retrieved from our off-chain Price Service API using the `pyth-evm-js` package.
         // See section "How Pyth Works on EVM Chains" below for more information.
```

Comment:
Might be worth removing the on-demand explanation from this page, and replacing it with a link to https://docs.pyth.network/consume-data/on-demand so we don't have duplicate explanations